.github/workflows/pkgci_regression_test_cpu.yml

```diff
@@ -71,14 +71,58 @@ jobs:
           pytest SHARK-TestSuite/iree_tests/onnx/ \
               -n auto -rpfE --timeout=30 --retries=2 --durations=20 \
               --config-files=build_tools/pkgci/external_test_suite/onnx_cpu_llvm_sync.json
-      # TODO(scotttodd): add a local cache for these large files to a persistent runner
+  # persistent runner with cache enabled for model testing
+  linux_x86_64_models:
+    name: Linux (x86_64) Model Testing/Benchmark
```

Comment:
I'm keeping an eye on workflow / job names. This is rather long:

`PkgCI / Regression Test CPU / Linux (x86_64) Model Testing/Benchmark (pull_request)`

May do a scrub through the various workflow files and pick new names at some point.